src/zeroD/ReactorNet.cpp

```diff
@@ -218,6 +283,45 @@ void ReactorNet::getState(double* y)
     }
 }
 
+void ReactorNet::getDerivative(int k, double* dky)
+{
+    double* cvode_dky = m_integ->derivative(m_time, k);
+    for (size_t j = 0; j < m_nv; j++) dky[j] = cvode_dky[j];
```

Comment:
All `for` loops should be multi-line, and get curly braces.

Reply:
OK. While not required, I was not following the established style. Will be fixed.